trade_remedies_public/config/settings/base.py

```diff
@@ -359,7 +360,10 @@
 COUNTRIES_FIRST = ["GB"]
 COUNTRIES_FIRST_BREAK = "------"
 
-FILE_MAX_SIZE_BYTES = 30000000
+FILE_MAX_SIZE_BYTES = 31457280
```

Comment:
I think should be an environment variable so we can test and change between environments going forward.

```suggestion
FILE_MAX_SIZE_BYTES = env.int("FILE_MAX_SIZE_BYTES", default=31457280)
```

Reply:
Resolved in https://github.com/uktrade/trade-remedies-public/pull/308/commits/0e4caf69cf2b42d20338a9a8e6781523b336904c.